portfolio/src/main/webapp/hobbies.html

```diff
@@ -0,0 +1,132 @@
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1>Hobbies</h1>
+
+    <h2>Sports</h2>
+
+    <div class = "hobby">
+      <h4>Tennis</h4>
+      <p>
+          Tennis is easily defined as my favorite sport. I started playing with my father when I
+          was around four years old. He would take my brother and I out to play even though we were
+          not able to properly hit the ball over the net yet. Over the years, what used to be an 
+          activity meant to bring us together as a family transitioned into an activity I am
+          deeply passionate about.
+      </p>
+      <p>
+          I joined my first team and got my first coach as a freshman in high school. I joined
+          varsity right away and played at that level for all four years. Throughout my high school
+          career I earned many league awards and finished first or second in the league for three
+          consecutive years.
+      </p>
+      <p>
+          I still love tennis to this day, and I will continue practicing to participate in local 
+          tournaments.
+      </p>
+    </div>
+
+    <div class = "hobby">
+	  <h4>Soccer</h4>
+	  <p>
+		  Before I fell in love with tennis, I enjoyed soccer. Prior to entering high school, I had no
+		  friends that played tennis. So, I directed my energy towards soccer.
+	  </p>
+      <p>
+	      I played for many years, with a brief period of inactivity due to health concerns, but I was
+	      never that good. I gradually started to enjoy soccer less and less, but I will always watch
+	      the FIFA World Cup. During the last tournament, Peru had finally qualified after 36 years of
+	      not making an appearance in the tournament. Vamos Perú!
+      </p>
+    </div>
+	
+    <div class = "hobby">
+	  <h4>Basketball</h4>
+      <p>
+		  It wasn't until my junior year of high school that I actually started to enjoy basketball.
+		  I wish I had shown interest in it earlier, but it wasn't until I saw friends of mine on
+		  the basketball team play that I was like, "WOW."
+      </p>
+      <p>
+	      Basketball is my favorite sport to watch and my favorite player is Luka Doncic. Even
+	      though Luka Doncic is my favorite player and I was raised in New York all my life, my
+	      favorite team is neither the Dallas Mavericks or a New York team (disappointing, I
+	      know). My favorite team is the Houston Rockets because James Harden made me a fan of
+	      the sport. The way he plays is just too fun to watch. Even though he still doesn't have
+	      a championship ring, I know he will get one before he retires.
+      </p>
+      <p>
+	      Given all that I have said, my prediction for this year's
+	      champion is the Los Angeles Lakers. If the break due to quarantine hadn't occurred I
+	      believe the Los Angeles Clippers were going to win, but now that Lebron James is well
+	      rested I know he will carry his team to another ring.
+      </p>
+    </div>
```

Comment:
```suggestion
    <div class = "hobby">
      <h4>Basketball</h4>
      <p>
        It wasn't until my junior year of high school that I actually started to enjoy basketball.
        I wish I had shown interest in it earlier, but it wasn't until I saw friends of mine on
        the basketball team play that I was like, "WOW."
      </p>
      <p>
        Basketball is my favorite sport to watch and my favorite player is Luka Doncic. Even
        though Luka Doncic is my favorite player and I was raised in New York all my life, my
        favorite team is neither the Dallas Mavericks or a New York team (disappointing, I
        know). My favorite team is the Houston Rockets because James Harden made me a fan of
        the sport. The way he plays is just too fun to watch. Even though he still doesn't have
        a championship ring, I know he will get one before he retires.
      </p>
      <p>
        Given all that I have said, my prediction for this year's
        champion is the Los Angeles Lakers. If the break due to quarantine hadn't occurred I
        believe the Los Angeles Clippers were going to win, but now that Lebron James is well
        rested I know he will carry his team to another ring.
      </p>
    </div>
```